src/App.js

```diff
@@ -26,10 +26,26 @@ class App extends Component {
         focusTime: 1500,
         shortBreakTime: 300,
         longBreakTime: 900,
-        sessionNumber: 1
+        sessionNumber: 1,
     };
+    this.changeSessionTime = this.changeSessionTime.bind(this);
+    this.changeBreakTime = this.changeBreakTime.bind(this);
   }
 
+  changeSessionTime(newTime) {
+    this.setState({
+        focusTime: newTime*60,
```

Comment:
variable name could have inferred if it was in seconds or minutes. Ideally this value should be in seconds